src/Services/Authentication/RemoteAppAuthenticationClaimsSerializer.cs

```diff
@@ -0,0 +1,36 @@
+using System;
+using System.IO;
+using System.Security.Claims;
+using System.Threading.Tasks;
+
+namespace Microsoft.AspNetCore.SystemWebAdapters.Authentication;
+
+internal class RemoteAppAuthenticationSerializer : IClaimsSerializer
```

Comment:
I don't see anything here that would be remote app specific. This seems like a general purpose claims principal serializer

Reply:
Good point. Maybe `BinaryClaimsSerializer` instead?